empyrical/stats.py

```diff
@@ -727,6 +731,46 @@ def tail_ratio(returns):
         np.abs(np.percentile(returns, 5))
 
 
+def cagr(returns, starting_value, period=DAILY, annualization=None):
+    """
+    Compute compound annual growth rate.
+
+    Parameters
+    ----------
+    returns : pd.Series
+        Daily returns of the strategy, noncumulative.
+        - See full explanation in :func:`~empyrical.stats.cum_returns`.
+    starting_value : float
+       The starting returns.
```

Comment:
Sorry, what I meant was, why would you ever pass a different `starting_value` to this function?